packages-new/components/menu/src/components/osds-menu/interfaces/attributes.ts

```diff
@@ -0,0 +1,8 @@
+interface OdsMenuAttributes {
```

Comment:
It was not mention in the doc, but we try to keep object singular (and only array/list in plural), would be nice to update to `OdsMenuAttribute`